lib/rules/rules.json

```diff
@@ -272,5 +272,9 @@
     "flatten": {
     	"compatible": true,
     	"alternative": "Array.prototype.reduce((a,b) => a.concat(b), [])"
+    },
+    "throttle": {
+        "compatible": true,
+        "alternative": "Example of native implementation: https://github.com/you-dont-need/You-Dont-Need-Lodash-Underscore#_throttle"
     }
-}
```

Comment:
Could you revert the change of this line? Otherwise LGTM. We should standardise link to the readme.